docker-compose.yml

```diff
@@ -1,9 +1,27 @@
-version: "3"
+version: "3.4"
```

Comment:
Hi, I was wondering why you're pinning to version `3.4` when the latest version is `3.8` is there a particular incompatibility?

Reply:
Hey there, no not all, happened to have a snippet of an example docker-compose file that seems to be outdated. :P 

I can try and update.

Reply:
Makes sense 👍